lm_eval/models/neuron_optimum.py

```diff
@@ -289,6 +288,7 @@ def __init__(
 
         self.vocab_size = self.tokenizer.vocab_size
         self.tokenizer.pad_token_id = self.tokenizer.eos_token_id
+        self.add_bos_token = self.add_bos_token
```

Comment:
@haileyschoelkopf is this really correct? seems like self-assignment 
`self.add_bos_token = add_bos_token` may be supposed?

Reply:
You're correct, good catch--mind PR'ing a fix for this?

Reply:
Yeah, i will do PR for that. And will try find if there are more similar typos.

Reply:
[Done](https://github.com/EleutherAI/lm-evaluation-harness/pull/1990)